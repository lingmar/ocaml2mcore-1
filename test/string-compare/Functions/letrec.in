let rec foo x =
  bar x
and bar x =
  42
in foo