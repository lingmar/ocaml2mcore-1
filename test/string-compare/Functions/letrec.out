mexpr
recursive
  let foo = lam x.
    bar x
  let bar = lam x.
    42
in foo
